controller_manager:
  ros__parameters:
    update_rate: 10
    use_sim_time: false

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    arm_controller:
      type: joint_trajectory_controller/JointTrajectoryController

arm_controller:
  ros__parameters:
    joints:
      - shoulder_pan
      - shoulder_lift
      - elbow_flex
      - wrist_flex
      - wrist_roll
      - gripper

    command_interfaces:
      - position

    state_interfaces:
      - position
      - velocity

    allow_nonzero_velocity_at_trajectory_end: false
